apps/arweave/src/ar_http_iface_middleware.erl

```diff
@@ -1555,7 +1555,7 @@ serve_format_2_html_data(Req, ContentType, TX) ->
 				{error, tx_data_too_big} ->
 					{400, #{}, jiffy:encode(#{ error => tx_data_too_big }), Req};
 				{error, not_found} ->
-					{200, #{ <<"content-type">> => ContentType }, <<>>, Req};
+					{404, #{}, <<"Not Found">>, Req};
```

Comment:
For consistency with the other `GET /tx` might be worth updating this line as well:
https://github.com/ArweaveTeam/arweave/pull/472/files#diff-0b7084531612365a22d52032a422eb664adc91b34cefb5f8d9499584072aa489R1526

Also, it looks like the `GET /tx/ID/data.html` endpoint uses this to report a 404:
https://github.com/ArweaveTeam/arweave/pull/472/files#diff-0b7084531612365a22d52032a422eb664adc91b34cefb5f8d9499584072aa489R334

`{404, #{}, sendfile("data/not_found.html"), Req};`